api/v1/storagecluster_types.go

```diff
@@ -92,6 +92,19 @@ type StorageClusterSpec struct {
 	// BackingStorageClasses is a list of storageclasses that will be used for the
 	// storageDeviceSets.
 	BackingStorageClasses []BackingStorageClass `json:"backingStorageClasses,omitempty"`
+	// DefaultStorageProfile is the default storage profile to use for
+	// the storageclassclaims as StorageProfile is optional.
+	DefaultStorageProfile string `json:"defaultStorageProfile,omitempty"`
+
+	StorageProfiles []StorageProfile `json:"storageProfiles,omitempty"`
+}
+
+// StorageProfile is the storage profile to use for the storageclassclaims.
+type StorageProfile struct {
+	// Name of the storage profile.
+	Name string `json:"name,omitempty"`
+	// DeviceClass is the deviceclass name.
+	DeviceClass string `json:"deviceClass,omitempty"`
```

Comment:
Didn't we discussed adding parameters to the storage profile?

Reply:
AFAIR We decided to have a separate struct for cephfs and rbd each will contains its parameters.

Reply:
Those section will be part of the profile, right?

Reply:
See correct, those are added in line 109-111